.github/workflows/ci-python.yml

```
@@ -0,0 +1,44 @@
+    strategy:
+      matrix:
+        python-version: ['3.10', '3.11', '3.12', '3.13', '3.14']
+      fail-fast: false
+
+    steps:
+      - uses: actions/checkout@v6
+
+      - name: Install uv
+        uses: astral-sh/setup-uv@08807647e7069bb48b6ef5acd8ec9567f424441b # v8.1.0
+        with:
+          python-version ${{ matrix.python-version }}
```

actionlint (1.7.12)

[error] "with" section is scalar node but mapping node is expected

(syntax-check)